tests/src/main/java/com/hazelcast/simulator/tests/replicatedmap/ReplicatedMapTest.java

```diff
@@ -91,12 +94,15 @@ public Worker createWorker() {
 
     private class Worker extends AbstractWorker<Operation> {
 
+        private final Metronome metronome = SimpleMetronome.withFixedIntervalMs(intervalMs);
```

Comment:
just a minor remark - current implementation of `Metronome` always uses busy-spinning.

it might not be that suitable when multiple tests are running in parallel. perhaps we could have another implementation when busy-spinning is undesired. CC @Donnerbart